### ISSUE TITLE
Remove PLACES options from Advanced Options

### ISSUE THREAD

**Issue by sisskind** [ngageoint/hootenanny-ui#764]
Should be completed along side ngageoint/hootenanny#1083.

**Comment by sisskind** [comment]
@bwitham is this ready to be worked on from UI side? (aka is ngageoint/hootenanny#1083 being worked on/can be worked on/done?)

**Comment by bwitham** [comment]
No one has worked on ngageoint/hootenanny#1083 yet.

**Comment by sisskind** [comment]
@bwitham would it be bad form to remove from UI (Advanced Options) before #1083 is worked on?

**Comment by bwitham** [comment]
No, I think that's fine.  I'm probably going to knock #1083 this week anyway.

**Comment by sisskind** [comment]
@bwitham I actually think this ticket isn't required anymore since you're removing from JSON which updates the UI.  If you concur, we can close.

**Comment by sisskind** [comment]
advanced_options feature
